src/content/reference/react/useState.md

```diff
@@ -4,7 +4,7 @@ title: useState
 
 <Intro>
 
-`useState` is a React Hook that lets you add a [state variable](/learn/state-a-components-memory) to your component.
+`useState` merupakan React Hook yang memungkinkan Anda untuk menambahkan [variabel *state*](/learn/state-a-components-memory) pada komponen anda.
```

Comment:
"Anda", always use "A" capital.

Reply:
[Reference](https://ejaan.kemdikbud.go.id/eyd/penggunaan-huruf/huruf-kapital/#23-huruf-kapital-digunakan-sebagai-huruf-pertama-kata-penunjuk-hubungan-kekerabatan-seperti-bapak-ibu-kakak-dan-adik-serta-kata-atau-ungkapan-lain-termasuk-unsur-bentuk-ulang-utuh-yang-digunakan-sebagai-sapaan):

![image](https://user-images.githubusercontent.com/6315466/236875185-8d26d367-1042-4f80-b5b6-836b9df305f2.png)

Reply:
Thank you for the reference!